e2e/tests/cloudformation.go

```diff
@@ -0,0 +1,39 @@
+package tests
+
+import (
+	"log"
+
+	"github.com/giantswarm/aws-operator/e2e/k8s"
+	"github.com/giantswarm/micrologger"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	cs, err := k8s.Client()
+	if err != nil {
+		log.Println("Could not create k8s client: ", err.Error())
+		return
+	}
+	logger, err := micrologger.New(micrologger.DefaultConfig())
+	if err != nil {
+		log.Println("Could not create logger: ", err.Error())
+		return
+	}
+
+	ts := &TestSet{
+		clientset: cs,
+		logger:    logger,
+	}
+	Add(ts.TestCRExists)
+}
+
+func (ts *TestSet) TestCRExists() (string, error) {
+	desc := "cloudformation TPR exists"
+
+	_, err := ts.clientset.ExtensionsV1beta1().ThirdPartyResources().Get("cloudformation.giantswarm.io", metav1.GetOptions{})
```

Comment:
I don't think this TPR is correct. The cloud formation resource will still process awstpr but it will do CRUD operations on the Cloud Formation stack.

Reply:
Ok, sorry missed that from the spec, I've modified it to check for awsptr being defined.